### DISCUSSION TITLE
童话故事《国王的新冠》

### DISCUSSION THREAD

**Discussion by fanhan-inside** [fanhan-inside/fanhan-inside.github.io#293] [General]
> “可是他头上戴的是绿帽子啊……”

闹剧结束了：

![](https://user-images.githubusercontent.com/41445691/215627774-0fccae5a-e8b8-4b84-8cf0-3972911a3160.png)

前情提要：

+ [“马莉娅同志在与我结婚时是纯洁的处女，婚后一贯正派⸺约瑟夫”](https://fanhan-inside.github.io/essay/modern/006.html)
+ [“1485年，那是一个夏啊啊啊天，有一只蝈蝻，戴上了绿帽摘下了王盎盎盎冠”](https://fanhan-inside.github.io/issues/0,000,140.html)

![](https://user-images.githubusercontent.com/41445691/215627778-fc27cee6-1f15-40e1-8e6c-834deb7e6ed9.png)

![](https://user-images.githubusercontent.com/41445691/215627781-03193e19-5d96-4985-a54d-771fe173439e.png)

![](https://user-images.githubusercontent.com/41445691/215627783-02b3e05b-6dd5-426d-8300-0ac77bd5b8be.png)

----

长话短说，之前日本宣布「准备在5月8日开始“just flu”政策」的经由权威机构认证的情报就没备份，因为预定今年五月加冕的是南不列颠及北爱尔兰联合王国的「查理三」，既不是「大日本帝国天皇」也不是「后清国太祖高皇帝」。

众所周知历任美国总统都有白金汉宫王室「血统」（家谱意义上的），包括奥巴马在内；而溥仪曾经代表爱新觉罗家与日本皇室联宗，通古斯民族权力机构和民族武装力量当中主事儿的「旗人」与日本那位法律上无「人」权的吉祥物之间也有家谱意义上的「血缘关系」。

在[《面对这帮记忆只有七秒钟的逗哔我也很为难》](https://fanhan-outside.vercel.app/bbs/mongolism.html)当中已经备份了说过好多遍但总是被一帮包衣阿哈胡搅蛮缠再被有关部门临时工党性坚定的老审查员以政治敏感理由亦当删去的经由权威机构认证的事实：溥杰的闺女已经归化成日本人了，有五个孩子，再招个姓啥的啥种族的「婿养子」接班然后登基称帝组建「满州国流亡政府」，并不奇怪。

眼下让我们先应景关注南不列颠及北爱尔兰吧，那边与《启示录》四季大戏的关系更加紧密一点，尤其是詹姆斯1611钦定本圣经，会被哪位自称「<del>太祖高皇帝</del><ins>小国王</ins>」的公众人物招牌活广告到处放风带节奏呢。

**Comment by fanhan-inside** [comment]
补充情报兼素材：

![](https://user-images.githubusercontent.com/41445691/215634963-ff205e32-18c5-4ecd-b44c-248bd07c3c5f.png)
![](https://user-images.githubusercontent.com/41445691/215635849-0758db59-3cb0-45d1-9676-be02f7a333ed.png)

前情提要：

+ [寓言故事：《旗人忧天》](https://fanhan-inside.github.io/issues/0,000,220.html)
+ [寓言故事：《俩哥布林辩月远伦敦近》](https://fanhan-inside.github.io/issues/0,000,204.html)

以及：

> J.van.Schooten 2022-3-26 17:04
> 
> 民主党传统风俗习惯就这样，这次是不是由于相对国力下降“玩脱”现在还不好说。中国和罗刹只有伙伴关系没有结盟，和北棒就有正式同盟条约。也就是说，在欧洲撺掇罗刹成功了，中国还能勉强保持距离置身事外，但是在亚洲如果撺掇北棒“试射”核弹还来个“误炸”……三战当然就打起来了也。
> 
> 民主党发家就是南方种植园主和西进运动时期的牛仔牧场主，都属于什么成分心里都有谱，洗白的是建制派（就是“老白男”），没洗白的就是3K党。没写错，看见德州风土人情当中“3K党攻击共和党”的震惊重磅突发号外大新闻用不着奇怪，老关系都还在。战后美国两党政治版图互换，这时民主党已经是以高学历精英老白男为首的黑帮联盟了，几十年整合下来才有现在的结果。美国民主党的形成过程类似古巴共产党那种“游击队联盟”（1959年就革命胜利直到1975年才召开“一大”制定党章）。
> 
> 就从市面上观察江湖好汉草莽豪杰地痞流氓泼皮无赖的一贯作风，也能判断出“三北防护林大汗”特色技战术“吹瓶伏特加光膀子硬刚军警宪特”并不常见，通常是“打一枪换一个地方”“敌进我退敌退我追敌驻我扰敌疲我打”。所以吹捧切・格瓦拉“流寇主义”（革命形势可以通过一小撮武装分子创造出来），市面上所有纪念“放把火就跑”“傻了吧爷会飞”系列重要游击战争的活动，差不多都是民主党赞助的。
> 
> 或问了，民主党扮猪吃老虎屡次得手的底气是什么？在最近两次美国大选前后已经有知情人士提到了：情报机构在民主党手里，看候选人顺眼（其实是建制派后台大佬谈妥）就借汝一用，看着不顺眼（台上傀儡不听话）就不配合。甭看川普大统领言必称MAGA反黑绿反雌权，那些红脖子的基层动员活动都不是他本人直属机构搞的。确切的说，川普早年是民主党，反复横跳（或曰球员被俱乐部转手）多次了，这次当共和党外援而已。
> 
> 情报活动什么风格想必都有印象，文艺作品当中那些窃取国家机密兼宇宙奥秘之类活动，是见得光的部分，哪怕暗杀抢劫破坏也是军事行动的一部分，观众都能理解。实际上主要活动是“撒谎偷窃诈骗，还开设专门课程传道授业解惑”（2019年4月15日，时任中统局长蓬佩奥于德州农工大学），**从业者通常走街串巷与城狐社鼠勾肩搭背称兄道弟推杯换盏**，接到指示之后才会发起各种恐怖袭击，并在内线的接应下胜利撤退。
> 
> 可以这么形象理解，在最近流行的文艺作品当中，这种“基层党员”及其外包的临时工，被称为“哥布林”，相关描述真是入木三分，相当贴切。
> 
> 补充内容2022-3-26 17:17
> 
> 上面有个笔误，蓬佩奥振聋发聩的演讲当时，是美国人民的好总理，之前曾担任中统局长。

相关内容「教员与塔利班」在之前已经谈笑风生过了，分散在各处，此处不再罗列。可以观察那些把大救星称为「教员」但是满口污言秽语的根红苗正忠君爱国童年才俊，通常还在吹捧「塔利班」，去年前年「阿富汗革命胜利」前后最为明显，其中存在着统计上的相关性。

如果联系自己的亲身经历，我目前四十四岁零两个月再两周还零一天，脱离权威机构教育部/工信部认证的真・学校已经二十多年了。而自称「教员」的家伙，参考家贼年龄应该七十多岁，五十几年前刚好是「塔利班」的身份，戴着红袖箍到处打砸抢抄抓。那么「革命家风代代传」之后，这种「清理三种人的漏网之鱼」货色还能手把手教出什么得意门生来，哪怕用汗毛思考也能判断。

有个文艺理论旁证，也是实体书记载，就是上世纪八十年代二十多岁的崔健开演唱会，下面铁杆粉丝都是四十多岁，听到《从头再来》歌词：「我不愿离开，我不愿存在，我不愿活得过分实实在在。我想要离开，我想要存在，**我想要死去之后从头再来**。」的时候，涕泪交流哇哇的哭。

为什么呢？有个民间段子：**刚出生，就挨饿；刚上学，就停课；刚毕业，就下乡；回到城里卖洋火。**

说的就是「老三届」学生。

**Comment by fanhan-inside** [comment]
既然提到情报活动了，那么顺便展开几句。先重复一遍革命导师和码字巨侠的谆谆教诲：

> 革命导师教导我们说
> 
> 当一个国家的政治以及经济出现重大危机时，爱国主义的破旗就又散发出臭味来。⸺革命导师圣列宁
> 
> 码字巨侠教导你们说
> 
> 爱国主义是流氓无赖们最后的藏身之地Patriotism is the last refuge of a scoundrel⸺塞缪尔・约翰逊

以前说过了，古拉格北里社区简直就是「小联合国」，什么人都有，什么标识都有。想必各位读者见过身着星条旗米字旗乃至其它国家和地区注册商标文化衫的家伙，除了「膏药旗」符号会遭遇口诛笔伐之外，其它都不会。

我还见过身着「出口转内销」的军用迷彩服和军靴的家伙，自称是「联合国维和部队」专用，只不过袖标上面也是星条旗米字旗，偶尔有几种三色旗（仏独露）。除此之外，我中华兲嘲上国「北京当局」专用的国旗国徽也见过，通常是「奥运同款队服」，有运动服有羽绒服还有短袖文化衫，台北/香港/澳门当局专用的没见过。

这些事情本来没啥，但是正如前一篇日志所述，如果多年来一直眉飞色舞讨论如何运用充沛人脉「薅国家羊毛」「挖社会主义墙角」还引以为荣的那些家伙⸺无论是葡萄架社区出行路巡逻员「污言秽语中年妇女」还是湘潭铺子门口站岗放哨的哥布林⸺本来都是与国际接轨的装扮，忽然有一天换成汉服唐装至少是充沛着我中华兲嘲上国传统装饰符号的服装，还佩戴国徽胸章国旗袖箍（虽然污言秽语和肆无忌惮的表现并未改变）……又是什么原因呢？

解读很简单，估计是遭遇「西贡铁拳」了，那些吹嘘的「“海里有人”“海边有人”“手眼通中央军委/政法委”」的人脉之上线已经跑了，于是忙不迭赶紧找个「庇护所」……黑色人渣只要表演「忠君爱国」，就能被红皮傻哔以为「根红苗正」。

![](https://user-images.githubusercontent.com/41445691/215647917-5d7e8028-d227-47bf-b637-3bec6b846dc9.png)

补充背景「三种人」：

+ 一是指“文化大革命”期间跟随林彪、江青一伙**造反起家、占据领导岗位**、干了坏事、情节严重的人；
+ 二是指帮派思想严重，在“文化大革命”期间宣扬林彪、“四人帮”的反动思想，**拉帮结派干坏事**，粉碎“四人帮”以后，立场观点没有转变，抵制中央的路线、方针、政策，**明里暗里继续进行帮派活动的人**；
+ 三是指“文化大革命”期间**挟嫌报复、行凶作恶、蓄意诬陷**，策划、指挥或亲自动手打砸机关、抢档案、破坏公私财物的主要分子，幕后策划或指挥武斗，造成严重后果的分子。

**Comment by fanhan-inside** [comment]
顺便补充点个人亲身经历的第一手材料作为旁证。

本来不想写这些，但是如同以前声明过的一样，如果没有在「草稿长编」当中按照「宁失于繁毋失于简」原则繁琐的「发牢骚」，很多没有亲身经历的读者就无法理解这些素材的运用方式。这次也一样，顺便还可以证明「无一字无来历」的职业习惯，为啥上面繁琐的补充背景还将情报中「其它塔利班正常教，逮住一个异类往死里整」的自称「革命导师」或「教员」的家伙称为「清理“三种人”时的漏网之鱼」。

以前提到过相关情况，在去古拉格南里社区住宅料理后事的时候，发现室内一片狼藉，那四分五裂的大号家具就不是家贼能干出来的重体力劳动，疑似有精壮「塔利班」入室打砸抢抄抓。但是居委会与人民警察都没有任何表示，似乎是按照「民不举官不究」方式忽略。

然后就是除了户口本之外，家贼的所有证件，包括北里住宅的房产证，都不翼而飞。而在最近办理各种手续的时候，家贼的**档案**又找不到了。虽然公开新闻显示北京市在2020年底就已经完成了「退休人员养老社会化的档案移交」工作，但是家贼的档案既不在人事关系所在的街道办事处，也不在人力资源公共服务中心。

并且家贼原单位「失联」的情况比较诡异。首先1月9日我直接去行政处和离退休办公室地址，大门紧锁，看锁头的情况有一阵子没人了。然后打电话，前几天还是连续响铃但没人接，而从1月13日开始的整个周末，都是「正在通话中」，半夜也这样，应该是有人**把话筒拿起放在一边**的表现。

由于类似情况（老国企退休职工档案失踪）比较多，并且我手头还有上一次（2013年家贼表演一路小跑追公交车那次）让我放弃继承权的公证书的存在，当时调档证明应该还有存根，所以家贼档案并非必须，总之其它材料足够于是公证处受理了。

这就是「三种人」定义当中「打砸、抢档案（以及证件）、破坏财物」条款所能覆盖的情况，至于「造反起家、占据领导岗位」「帮派思想、拉帮结派、帮派活动」参考其它材料，此处不再重复。

**Comment by MAO-da** [comment]
范队最近还好么，二月和三月未见你更新

**Reply by fanhan-inside** [reply]
不好，脑桥出血，头昏眼花半身不遂，最近才有机会/能力独眼单手更新个人博客站点。

**Reply by CESTBON5146** [reply]
脑桥出血已经很严重了，望保重
